Depth_First_Search/dps.cpp

```diff
@@ -0,0 +1,58 @@
+#include<iostream>
+#include<list>
+using namespce std;
+class Graph
+{
+  int V;
+  list<int>* adj;
+  void DFSUtil(int v, bool visited[]);
+public:
+  Graph(int v);
+  void addEdge(int v,int w);
+  void DFS(int v);
+}
```

Comment:
} --> };
class should be inbound semi-colon at then end of curly braces.
Thank You.

Reply:
Thanks for the suggestions @starkblaze01.
I have made the required changes.